src/pythermalcomfort/models.py

```diff
@@ -1069,7 +1071,7 @@ def utci(tdb, tr, v, rh, units="SI", return_stress_category=False):
     if units.lower() == "ip":
         tdb, tr, v = units_converter(tdb=tdb, tr=tr, v=v)
 
-    check_standard_compliance(standard="utci", tdb=tdb, tr=tr, v=v)
```

Comment:
I have noticed that you removed this check. How can we return a warning if the inputs are outside the UTCI model limits of applicability?

Reply:
I noticed that the standard compliance test for `utci` here:
https://github.com/CenterForTheBuiltEnvironment/pythermalcomfort/blob/fc143fe26637765518d97dcdd93d9fb821d56d89/src/pythermalcomfort/utilities.py#L27-L33
raises a warning for wind speed. However, i noticed that this was checked again anyway:
https://github.com/CenterForTheBuiltEnvironment/pythermalcomfort/blob/fc143fe26637765518d97dcdd93d9fb821d56d89/src/pythermalcomfort/models.py#L1093-L1100

The warning is just a fundamental difference if we have arrays here, i guess? If i have a large array, e.g., a large geospatial dataset like ERA5 as suggested here #2 , there might be valid and invalid input combination at the same time and you don't want to throw millions of warnings as you would have for a geospatial dataset. So instead, i suggest to return `np.nan` instead of raising a warning or the `ValueError`.

Long story short: I am not sure, if we would break anything for users if the function returns `np.nan` instead of raising a `ValueError` (although most of the time going on with nan should also raise an error...) For now, i could introduce a check, if the input is just a single value to still raise an error and a warning that this might change in the future, what do you think?

Reply:
i noticed that there was no test for the case a users input was invalid....

Reply:
One example of how returning `nan` instead of raising a warning may be a problem is presented below. 

The ASHRAE and ISO have slightly different applicability limits for the PMV model. If we return a warning and we still calculate a PMV value nothing will break in our users' code. If on the other hand, we return a `nan` then their code will break. I know that most people ignore warnings and returning a `nan` is more effective and informative, but at the same time, for example, users will find it strange that for example for a `tdb = 49.9` we return a UTCI value while for  `tdb = 50.1` we do not.

What are your thoughts on this?

Reply:
You are right there was no test. I did not implement all the necessary tests. More tests are welcome :)

Reply:
A few options on how to solve this issue are below:
* we return `nan` only if the user inputs arrays, while we raise a warning if the user inputs a single value and we clearly document that.
* we calculate a UTCI value even if the input is outside the model limit, but we also return an array with boolean values listing for each element if the inputs exceeded the applicability limits.

Perhaps the second solution is a bit better. I am open to other suggestions.

Reply:
i would also prefer the second solution, maybe even in the form of an argument like `ignore_limits`, so if that's true, also values outside of the applicability limits will be returned, otherwise `np.nan`. By default, it should be `False`.

Reply:
> One example of how returning `nan` instead of raising a warning may be a problem is presented below.
> 
> The ASHRAE and ISO have slightly different applicability limits for the PMV model. If we return a warning and we still calculate a PMV value nothing will break in our users' code. If on the other hand, we return a `nan` then their code will break. I know that most people ignore warnings and returning a `nan` is more effective and informative, but at the same time, for example, users will find it strange that for example for a `tdb = 49.9` we return a UTCI value while for `tdb = 50.1` we do not.
> 
> What are your thoughts on this?

at least, for `utci` also for invalid float input, there would be an error, so returning `np.nan` instead would make sure, that user's code still breakes if they didn't take care of the `ValueError` before...

Reply:
> i would also prefer the second solution, maybe even in the form of an argument like `ignore_limits`, so if that's true, also values outside of the applicability limits will be returned, otherwise `np.nan`. By default, it should be `False`.

I agree, let's then use the second solution and add an input argument `ignore_limits`. Would you be able to implement the changes?

Reply:
yes, no problem!

Reply:
Let me know once you are done so I can accept your pull request and release a new version of pythermalcomfort. Thank you again for working on this. You are the first person from the community who contributes to the package :)